pages/linux/bully.md

```diff
@@ -0,0 +1,13 @@
+# bully
+
+> Used to bruteforce WPS pin of a wireless access point.
+> Necessary information must be gathered with `airmon-ng` and `airodump-ng` before using `bully`.
+> More information: <https://www.kali.org/tools/bully/>.
+
+- Display help information:
+
+`bully --help`
+
+- Crack the password:
+
+`bully --bssid {{mac}} --channel {{channel}} --bruteforce {{interface}}`
```

Comment:
Shouldn't it be `--channel N {{channel}}` (How about mentioning the number in the placeholder instead?

Ref: `-c, --channel N[,N...] : Channel number of AP, or list to hop [b/g]`

Reply:
It's better if someone can assist here 😄